src/helpers.coffee

```diff
@@ -2,6 +2,23 @@
 # the **Lexer**, **Rewriter**, and the **Nodes**. Merge objects, flatten
 # arrays, count characters, that sort of thing.
 
+# async/await for test-suite
```

Comment:
So what’s going on here? You’re polyfilling `async`/`await` for the purposes of the tests?